src/Files.App/Utils/Cloud/CloudProviders.cs

```diff
@@ -45,6 +45,8 @@ public enum CloudProviders
 
 		LucidLink,
 
-		kDrive
+		kDrive,
+
+		Sync
```

Comment:
```suggestion
		syncCom
```

Reply:
```suggestion
		SyncDrive
```
I think the above makes more sense.